client/state/plugins/wporg/actions.js

```diff
@@ -41,6 +58,60 @@ export default {
 					error: error
 				} );
 			} );
-		}
-	}
+		};
+	},
+
+	fetchPluginsList: function( category, page, searchTerm ) {
+		return debounce( ( dispatch ) => {
```

Comment:
let's be very careful here with debounce. let's add it when we realize we need it instead of in advance.

did you test to make sure this works? it looks like we won't actually be debouncing anything…

```js
const f1 = fetchPluginsList( 'myCategory', 1, 'splines' ) // === debounced thunk
const f2 = fetchPluginsList( 'myCategory', 1, 'splines' ) // === debounced thunk

f1 === f2 // false - we have to separate debounced functions, thus no debouncing
```